js/controller.js

```diff
@@ -193,7 +193,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       if (this.state.screenToShow !== CONSTANTS.SCREEN.AD_SCREEN ) {
         this.state.duration = duration;
         if (this.skin.props.skinConfig.upNext.showUpNext) {
-          if (!Utils.isIPhone()){//no UpNext for iPhone
+          if (!(Utils.isIPhone() || (Utils.isIos && this.state.fullscreen))){//no UpNext for iPhone or fullscreen iPad
```

Comment:
Missing parens at Utils.isIos, this will check if the function exists and not if it's iOS.